### ISSUE TITLE
Key:diaper has been deprecated

### ISSUE THREAD

**Issue by ValorNaram** [simonpoole/beautified-JOSM-preset#150]
Hello,

the key described at https://wiki.openstreetmap.org/wiki/Key:diaper has been superseded by https://wiki.openstreetmap.org/wiki/Key:changing_table

I suggest you supporting both keys while the old one is still in use.

**Comment by simonpoole** [comment]
@ValorNaram the main problem is that there is no way to deprecate attribute tags, so it essentially means allowing both in parallel.

See also https://github.com/MarcusWolschon/osmeditor4android/issues/969

**Comment by simonpoole** [comment]
changing_table etc added in https://github.com/simonpoole/beautified-JOSM-preset/commit/fb825ce0ae0644f8fb03d4ec7c02fe4bb4a214b5
